loadtest/control/simplecontroller/actions.go

```diff
@@ -149,6 +150,38 @@ func (c *SimpleController) searchUsers() control.UserStatus {
 	return c.newInfoStatus(fmt.Sprintf("found %d users", len(users)))
 }
 
+func (c *SimpleController) updateProfile() control.UserStatus {
+	userId := c.user.Store().Id()
+	userName := fmt.Sprintf("testuserNew%d", c.id)
+	nickName := fmt.Sprintf("testNickName%d", c.id)
+	firstName := fmt.Sprintf("firstName%d", c.id)
+	lastName := fmt.Sprintf("lastName%d", c.id)
+	err := c.user.PatchUser(userId, &model.UserPatch{
+		Username:  &userName,
+		Nickname:  &nickName,
+		FirstName: &firstName,
+		LastName:  &lastName,
+	})
+	if err != nil {
+		return c.newErrorStatus(err)
+	}
+	return c.newInfoStatus("user patched")
+}
+
+func (c *SimpleController) updateProfileImage() control.UserStatus {
+	// TODO: take this from the config later.
+	imagePath := "./testdata/test.png"
```

Comment:
I would maybe change the filename to `test_profile.png` since we might have other images of different sizes/formats to test other things like file uploads.